website/src/pages/index.js

```diff
@@ -7,56 +7,33 @@
 
 const React = require('react');
 import Layout from "@theme/Layout";
-import HomeSplash from '../components/HomeSplash';
-import Api from '../../data/features/api.mdx'
-import Bridging from '../../data/features/bridging.mdx'
-import Intents from '../../data/features/intents.mdx'
-import Directory from '../../data/features/app-directory.mdx'
-import ContextData from '../../data/features/context-data.mdx'
-import GridBlock from "../components/GridBlock";
-import Container from '../components/Container';
-import UseCases from '../../data/front/use-cases.mdx'
-import Conformance from '../../data/front/conformance.mdx'
-import Feature from '../components/Feature'
-import Showcase from "../../core/Showcase";
 
-import users from './../../data/users.json';
-
-
-const UserShowcase = () => {
-
-  const pinnedUsers = users.filter(user => user.pinned);
-  pinnedUsers.sort((a, b) => a.caption.localeCompare(b.caption))
-
-  return (
-    <div className="userShowcase productShowcaseSection padding-top--lg padding-bottom--lg" style={{textAlign: 'center'}}>
-      <h2>Who is Using FDC3?</h2>
-      <p style={{margin: 'auto'}}>The Financial Desktop Connectivity and Collaboration Consortium (FDC3) standards are created and used by <a href="/users">leading organizations across the financial industry</a>. For more detail on who's using FDC3, developer tools, training and examples see the <a href="/community">community page</a>.</p>
-      <Showcase users={pinnedUsers} />
-    </div>
-  );
-};
+import NewSplashTop from "../components/NewSplashTop";
+import UseCases from "../components/UseCases";
+import Benefits from "../components/Benefits";
+import Parts from "../components/Parts";
+import WhatIsIt from "../components/WhatIsIt";
 
+import UserShowcase from "../components/UserShowcase";
+import Conformance from "../components/Conformance";
+import Training from "../components/Training"
+import GetInvolved from "../components/GetInvolved"
 
 
 export default () => {
-  return (<Layout>
-    <HomeSplash />
-    <Container>
-      <GridBlock items={[
-        <Api />,
-        <Intents />,
-        <ContextData />,
-        <Directory />,
-        <Bridging />
-      ]} />
-    </Container>
-      <Feature>
-        <UseCases />
-      </Feature>
-      <UserShowcase />
-      <Feature>
-        <Conformance />
-      </Feature>
-  </Layout>)
+	return (<Layout>
+		<NewSplashTop />
+		<WhatIsIt />
+		<Benefits />
```

Comment:
I noticed that Benefits (Manage The Information Overload) and Parts (The API) sections both have a 1 above them in the output.  Did you want each of the sections numbered and it isn't being incremented, or do the numbers for the sections need to be removed?

![image](https://github.com/finos/FDC3/assets/28159742/797ec7ce-a2ad-4b47-9bf0-a05e1073a425)
![image](https://github.com/finos/FDC3/assets/28159742/f8adc850-3f44-4877-91a6-0934539571db)

Reply:
Each of the carousels have a number at the top so you can see where you're up to.  When you click prev/next the number changes